Common/include/config_structure.hpp

```diff
@@ -2872,6 +2885,48 @@ class CConfig {
 	 * \return Number of implicit smoothing iterations.
 	 */
 	unsigned short GetMG_CorrecSmooth(unsigned short val_mesh);
+  
+  /*!
+   * \brief Get the number of pre-smoothings in a multigrid strategy.
```

Comment:
Oops, you are right. Fixing the comments about FFD in the config file right now.